style.css

```
@@ -99,8 +101,8 @@ h2 {
 	display: flex;
 	justify-content: center;
 	align-items: center;
-	width: 300px;
-	height: 50px;
+	width: 100%;
+	padding: 16px 16px;
```

```suggestion
	padding: 16px;
```

Jeśli powielasz takie same wartości, to możesz napisać je w ten sposób, wyjdzie na to samo ale mniej kodu